### PR TITLE
Output files mod

### PR DESCRIPTION
**Pull request by asedlmayr**
Modified the input / output file paths and fixed a few minor errors

### CODE REVIEW

File: docker/run_both.sh [hunk 1]
```diff
@@ -8,21 +8,22 @@
 # Before we do anything, we need to add the proper env vars for Intel MKL, etc.
```
Comment (rdoosty) [inline docker/run_both.sh:8]: what is both?

**Review by rdoosty — APPROVED** [review]

File: docker/run_both.sh (hunk 1 above)
Comment (asedlmayr) [reply docker/run_both.sh:8]: Not clue what any of these do.  Both might be sender + agora

File: src/common/loggers/csv_logger.cc [hunk 2]
```diff
@@ -21,6 +23,9 @@ CsvLogger::CsvLogger(size_t log_id, const std::string& radio_name) {
   if (log_id >= kAllLogs) {
     AGORA_LOG_ERROR("Invalid log id %zu in CsvLogger\n", log_id);
   } else {
+    if (std::filesystem::is_directory("files/log") == false) {
```
Comment (di-mu) [inline src/common/loggers/csv_logger.cc:26]: spdlog creates the directory automatically if not exists. No need to create the directory manually.

File: files/config/examples/dl-hw.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
```
[the remark this answers is not in the document]
Comment (asedlmayr) [reply files/config/examples/dl-hw.json:1]: Not sure, we can remove it.   But probably should condense all the "powder / vulture / etc" versions

File: src/common/loggers/csv_logger.cc [hunk 4]
```diff
@@ -21,6 +23,9 @@ CsvLogger::CsvLogger(size_t log_id, const std::string& radio_name) {
...
+      std::filesystem::create_directory("files/log");
+    }
     const std::string filename =
         "log/log-" + kCsvName.at(log_id) + "-" + radio_name + ".csv";
```
Comment (di-mu) [inline src/common/loggers/csv_logger.cc:30]: I think it should now be "files/log/log-" + ...

**Review by di-mu — APPROVED** [review]

**Pull request merged**